flink-ml-python/pyflink/ml/lib/feature/tests/test_robustscaler.py

```diff
@@ -0,0 +1,135 @@
+from pyflink.ml.core.linalg import Vectors, DenseVectorTypeInfo, DenseVector
+
+from pyflink.ml.lib.feature.robustscaler import RobustScaler
+from pyflink.table import Table
+
+
+class RobustScalerTest(PyFlinkMLTestCase):
+
+    def setUp(self):
+        super(RobustScalerTest, self).setUp()
+        self.train_table = self.t_env.from_data_stream(
+            self.env.from_collection([
+                (1, Vectors.dense(0.0, 0.0),),
+                (2, Vectors.dense(1.0, -1.0),),
+                (3, Vectors.dense(2.0, -2.0),),
+                (4, Vectors.dense(3.0, -3.0),),
+                (5, Vectors.dense(4.0, -4.0),),
+                (6, Vectors.dense(5.0, -5.0),),
+                (7, Vectors.dense(6.0, -6.0),),
+                (8, Vectors.dense(7.0, -7.0),),
+                (9, Vectors.dense(8.0, -8.0),),
+            ],
+                type_info=Types.ROW_NAMED(
+                    ['id', 'input'],
+                    [Types.INT(), DenseVectorTypeInfo()])
+            ))
+
+        self.predict_table = self.t_env.from_data_stream(
+            self.env.from_collection([
+                (Vectors.dense(3.0, -3.0),),
+                (Vectors.dense(6.0, -6.0),),
+                (Vectors.dense(99.0, -99.0),),
+            ],
+                type_info=Types.ROW_NAMED(
+                    ['input'],
+                    [DenseVectorTypeInfo()])
+            ))
+
+        self.expected_output = [
+            Vectors.dense(0.75, -0.75),
+            Vectors.dense(1.5, -1.5),
+            Vectors.dense(24.75, -24.75)]
+
+    def test_param(self):
+        robust_scaler = RobustScaler()
+        self.assertEqual("input", robust_scaler.input_col)
+        self.assertEqual("output", robust_scaler.output_col)
+        self.assertEqual(0.25, robust_scaler.lower)
+        self.assertEqual(0.75, robust_scaler.upper)
+        self.assertEqual(0.001, robust_scaler.relative_error)
+        self.assertFalse(robust_scaler.with_centering)
+        self.assertTrue(robust_scaler.with_scaling)
+
+        robust_scaler\
+            .set_input_col("test_input")\
+            .set_output_col("test_output")\
+            .set_lower(0.1)\
+            .set_upper(0.9)\
+            .set_relative_error(0.01)\
+            .set_with_centering(True)\
+            .set_with_scaling(False)
+
+        self.assertEqual("test_input", robust_scaler.input_col)
+        self.assertEqual("test_output", robust_scaler.output_col)
+        self.assertEqual(0.1, robust_scaler.lower)
+        self.assertEqual(0.9, robust_scaler.upper)
+        self.assertEqual(0.01, robust_scaler.relative_error)
+        self.assertTrue(robust_scaler.with_centering)
+        self.assertFalse(robust_scaler.with_scaling)
+
+    def test_output_schema(self):
+        robust_scaler = RobustScaler()
+        model = robust_scaler.fit(self.train_table)
+        output = model.transform(self.predict_table.alias('test_input'))[0]
+        self.assertEqual(
+            ['test_input', 'output'],
```

Comment:
Let's test non-default output column names as well, and use `set_input_col` and `set_output_col` to specify the non-default values.